mypy/typeanal.py

```diff
@@ -1475,6 +1475,23 @@ def visit_callable_type(self, t: CallableType) -> TypeVarLikeList:
             return []
 
 
+def maybe_expand_unimported_type_becomes_any(
```

Comment:
TBH this name is confusing, `unimported_type_becomes_any_expanded` would be better IMO. Also, I am not sure this is a best module for this helper. Shouldn't it be in `messages.py`?